docs/src/pages/ja/getting-started.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: はじめに
+lang: ja
+---
+
+Astro（アストロ）は、モダンな静的サイトジェネレーターです。Astroについては、[ホームページ](https://astro.build/)や[リリース記事](https://astro.build/blog/introducing-astro)をご覧ください。このページでは、Astroのドキュメントおよび関連リソースの概要を紹介します。
+
+Astroの簡単な概要を知りたい方は[ホームページ](https://astro.build/)をご覧ください。
+
+## Astroを試す
+
+もっとも簡単なAstroを試す方法は、あなたのマシンの新しいディレクトリで `npm init astro` を実行することです。新しいAstroプロジェクトを立ち上げる際には、CLIウィザードがサポートしてくれます。
+
+5つの簡単なステップでAstroを使い始めるには、[クイックスタートガイド](/quick-start)をご覧ください。
+
+また、[インストールガイド](/installation)では、Astroのセットアップ方法を詳しく解説しています。
+
+## サンプルプロジェクト
+
+サンプルを使ってAstroを学びたい方は、GitHubにある[全サンプル](https://github.com/snowpackjs/astro/tree/main/examples)をご覧ください。
+
+これらのサンプルは、`--template` というCLIフラグを付けて `npm init astro` を実行すると、ローカルマシンでチェックアウトできます。また、 `--template` フラグは、サードパーティのコミュニティテンプレートもサポートしています。
+
+```bash
+# 公式テンプレートを使用して、initウィザードを実行
+npm init astro -- --template [OFFICIAL_EXAMPLE_NAME].
+# コミュニティテンプレートを使用して、initウィザードを実行
+npm init astro -- --template [GITHUB_USER]/[REPO_NAME]を実行します。
+npm init astro -- --template [GITHUB_USER]/[REPO_NAME]/path/to/example
+```
+
+### オンラインコードエディター
+
+ブラウザでAstroを試してみたいという方は、オンラインコードエディターを利用できます。[CodeSandboxの「Hello World!」テンプレート](https://codesandbox.io/s/astro-template-hugb3)を試してみてください。
+
+注：CodeSandboxでは、一部の機能（例：Fast Refresh）が現在制限されています。
```

Comment:
This line was updated two days ago. Please change it:)
 ref: [commit](https://github.com/snowpackjs/astro/commit/b0a7e79bd0d7a22e714b9ab22264eacaf746c8ad#diff-fa51230395c29de4ba2e8712db577a1a692c065209ed1c01e99c0b0576f0c607)